projects/plugins/protect/src/js/routes/settings/index.jsx

```diff
@@ -0,0 +1,171 @@
+import useAccountProtectionQuery from '../../data/account-protection/use-account-protection-query';
+import useToggleAccountProtectionMutation from '../../data/account-protection/use-toggle-account-protection-module-mutation';
+import useAnalyticsTracks from '../../hooks/use-analytics-tracks';
+import usePlan from '../../hooks/use-plan';
+import styles from './styles.module.scss';
+
+const SettingsPage = () => {
+	const SUPPORT_LINK = 'https://jetpack.com/?post_type=jetpack_support&p=324199';
+
+	const { hasPlan } = usePlan();
+	const { data: accountProtection } = useAccountProtectionQuery();
+	const toggleAccountProtectionMutation = useToggleAccountProtectionMutation();
+
+	/**
+	 * Toggle Account Protect Module
+	 *
+	 * Flips the switch on the Account Protection module, and then refreshes the data.
+	 */
+	const toggleAccountProtection = useCallback( async () => {
+		toggleAccountProtectionMutation.mutate();
+	}, [ toggleAccountProtectionMutation ] );
+
+	// Track view for Protect Account Protection page.
+	useAnalyticsTracks( {
+		pageViewEventName: 'protect_account_protection',
+		pageViewEventProperties: {
+			has_plan: hasPlan,
+		},
+	} );
+
+	const accountProtectionSettings = (
+		<div className={ styles[ 'toggle-section' ] }>
+			<div className={ styles[ 'toggle-section__control' ] }>
+				<ToggleControl
+					checked={
+						accountProtection.isSupported &&
+						! accountProtection.hasUnsupportedJetpackVersion &&
+						accountProtection.isEnabled
+					}
+					onChange={ toggleAccountProtection }
+					disabled={
+						! accountProtection.isSupported ||
+						accountProtection.hasUnsupportedJetpackVersion ||
+						toggleAccountProtectionMutation.isPending
+					}
+				/>
+			</div>
+			<div className={ styles[ 'toggle-section__content' ] }>
+				<Text variant="title-medium">{ __( 'Account protection', 'jetpack-protect' ) }</Text>
+				{ ! accountProtection.isSupported && (
+					<Notice
+						level="warning"
+						hideCloseButton={ true }
+						className={ styles[ 'toggle-section__alert' ] }
+						title={
+							<Text>
+								{ __(
+									'This feature has been disabled by your site administrator or hosting provider.',
+									'jetpack-protect'
+								) }
+							</Text>
+						}
+						actions={ [
+							<Button
+								variant="link"
+								isExternalLink
+								href={ SUPPORT_LINK + '#unsupported-environments' }
+								key="learn-more"
+							>
+								{ __( 'Learn more', 'jetpack-protect' ) }
+							</Button>,
+						] }
+					/>
+				) }
+				{ accountProtection.isSupported && accountProtection.hasUnsupportedJetpackVersion && (
+					<Notice
+						level="warning"
+						hideCloseButton={ true }
+						className={ styles[ 'toggle-section__alert' ] }
+						title={
+							<Text>
+								{ __(
+									'This feature has been disabled because Jetpack Protect is installed with an unsupported version of Jetpack. Please update Jetpack to version 14.5 or later to enable this feature.',
```

Comment:
I wonder if it would be worth adding the words "plugin" in there, so folks understand we're talking about running the Jetpack Protect plugin along with an old version of the Jetpack plugin. I think that may be obvious for us, but not that obvious for others.

Reply:
Addressed in: https://github.com/Automattic/jetpack/pull/42559